src/Catalog/Monitoring/Model/PackageTimestampMetadata.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NuGet.Protocol.Core.Types;
+using Newtonsoft.Json.Linq;
+using NuGet.Services.Metadata.Catalog.Helpers;
+using Microsoft.Extensions.Logging;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring.Providers
+{
+    /// <summary>
+    /// Represents the timestamp metadata for a single package in a package source.
+    /// </summary>
+    public class PackageTimestampMetadata : INuGetResource
+    {
+        public static PackageTimestampMetadata CreateExisting(DateTime created, DateTime lastEdited)
```

Comment:
What do you suggest? I'll add XML docs.

`CreateExisting` is for creating a `PackageTimestampMetadata` that represents a package that exists on a feed.

`CreateMissing` is for creating a `PackageTimestampMetadata` that represents a package that is missing from a feed.